solr/test-framework/src/java/org/apache/solr/embedded/JettySolrRunner.java

```diff
@@ -699,12 +653,6 @@ public synchronized void stop() throws Exception {
         timeout.waitFor("Timeout waiting for reserved executor to stop.", rte::isStopped);
       }
 
-      // we want to shutdown outside of jetty cutting us off
```

Comment:
as I said in the commit message for this change, this block here has been pointless ever since the previous big change by Gus.  The comment dates back to @markrmiller  but it's not clear to me way way back then what the concern was.  Any way, I've found Jetty to do stuff fairly predictably and in one thread that I can reason about the sequencing.